api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -62,8 +62,9 @@ const (
 
 // FelixConfigurationSpec contains the values of the Felix configuration.
 type FelixConfigurationSpec struct {
-	UseInternalDataplaneDriver *bool  `json:"useInternalDataplaneDriver,omitempty"`
-	DataplaneDriver            string `json:"dataplaneDriver,omitempty"`
+	UseInternalDataplaneDriver *bool            `json:"useInternalDataplaneDriver,omitempty"`
+	DataplaneDriver            string           `json:"dataplaneDriver,omitempty"`
+	DataplaneWatchdogTimeout   *metav1.Duration `json:"dataplaneWatchdogTimeout,omitempty" configv1timescale:"seconds"`
```

Comment:
Should this be commented?  (Obviously most config fields are.  But perhaps this initial group is special in some way?)